src/style.css

```diff
@@ -1 +1,67 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header_logo {
+  width: 40px;
+  height: 40px;
+  cursor: pointer;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav_list {
+  display: flex;
+  height: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+.nav_item {
+  list-style: none;
+  margin: 0 10px;
```

Comment:
use either margin-right or margin-left, and also one of the elements shouldn't have this margin